### DISCUSSION TITLE
Build and run the app using Docker

### DISCUSSION THREAD

**Discussion by JohnTan38** [pathwaycom/llm-app#73] [Q&A] [answered]
Hello Pathwaycom
We have cloned the repository, created '.env' file, and in VS code terminal execute 'docker compose run --build --rm -p 8080:8080 llm-app-examples'
Error message: /usr/bin/env: 'python\r': No such file or directory
/usr/bin/env: use -[v]S to pass options in shebang lines

The container could not be started in Docker. Please help to advise and assist to resolve this. Thank you very much

John Tan
john.tan@sh-cogent.com.sg

**Comment by dxtrous** [comment, marked as answer]
Hello @JohnTan38, the issue is one resulting from your Windows setup. The character `\r` that you see displayed in your error messages is "carriage return", which means the Unix subsystem/layer you are using does not handle the Windows end-of-line character correctly. 

The easiest fix is to save the affected files (docker files, etc.) in text editors which allow you to configure the end-of-line character, and make sure it is set to Unix.
